KeyConfig.h

```diff
@@ -43,6 +43,7 @@ class KeyConfig
         ACTION_CROP_VIDEO = 34,
         ACTION_PAUSE = 35,
         ACTION_PLAY = 36,
+	ACTION_CHANGE_FILE = 37,
```

Comment:
Indentation looks off

Reply:
Yep. It's corrected in the next pull request, based on this one. If that is ok otherwise, you may use that. Otherwise I will do a correction on this one...